### PR TITLE
fix: Layer sets focus visible classname for its FocusRectsProvider

### PR DESCRIPTION
**Pull request by smhigley**
## Previous Behavior

Focus outlines would not show up when opening a Modal, Dialog, Panel, etc. in a new Layer, since the layer contents would be under a new `FocusRectsProvider`.

## New Behavior

Layer now checks whether the focus visible classname is present in its own context, then sets it (if applicable) on the new provider it instantiates.

## Related Issue(s)

- Fixes [15778](https://dev.azure.com/microsoftdesign/fluent-ui/_workitems/edit/15778)

### CODE REVIEW

File: packages/react/src/components/Layer/Layer.base.tsx [hunk 1]
```diff
@@ -24,6 +26,14 @@ import type { ILayerProps, ILayerStyleProps, ILayerStyles } from './Layer.types'
 
 const getClassNames = classNamesFunction<ILayerStyleProps, ILayerStyles>();
 
+const getFocusVisibility = (providerRef?: React.RefObject<HTMLElement>) => {
+  if (providerRef?.current) {
+    return providerRef.current.classList.contains(IsFocusVisibleClassName);
+  }
+
+  return false;
+};
```
Comment (smhigley) [inline packages/react/src/components/Layer/Layer.base.tsx:35]: There is no focus visible state saved anywhere in Fluent -- it's done purely through setting the className based on user events. The only way to check the current state is to check for the presence of the className.

**Comment by fabricteam** [comment]
## 📊 Bundle size report

<sub>🤖 This report was generated against <a href='https://github.com/microsoft/fluentui/commit/7eca299aed541ae297ab5bc7d511e64f56570932'>7eca299aed541ae297ab5bc7d511e64f56570932</a></sub>
<div id="bundleSizeComment9423"/>

**Comment by fabricteam** [comment]
## 🕵 fluentuiv8 No visual regressions between this PR and main
<div id="vrtCommentfluentuiv8"/>

**Review by micahgodbolt — APPROVED** [review]

**Comment by fabricteam** [comment]
## [Perf Analysis (`@fluentui/react`)](https://github.com/microsoft/fluentui/wiki/Perf-Testing)
<p>No significant results to display.</p><details><summary>All results</summary><p>
  <table>
  <tr>
    <th>Scenario</th>
    <th>Render type</th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">Master Ticks</a>
    </th>
    <th>
      <a href="https://github.com/microsoft/fluentui/wiki/Perf-Testing#why-are-results-listed-in-ticks-instead-of-time-units">PR Ticks</a>
    </th>
    <th>Iterations</th>
    <th>Status</th>
  </tr><tr>
            <td>BaseButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/BaseButton-mount_base.html">629</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/BaseButton-mount.html">638</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Breadcrumb</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Breadcrumb-mount_base.html">1642</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Breadcrumb-mount.html">1661</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Checkbox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Checkbox-mount_base.html">1686</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Checkbox-mount.html">1677</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>CheckboxBase</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/CheckboxBase-mount_base.html">1446</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/CheckboxBase-mount.html">1459</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ChoiceGroup</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/ChoiceGroup-mount_base.html">2890</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/ChoiceGroup-mount.html">2925</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ComboBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/ComboBox-mount_base.html">648</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/ComboBox-mount.html">643</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>CommandBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/CommandBar-mount_base.html">6156</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/CommandBar-mount.html">6164</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>ContextualMenu</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/ContextualMenu-mount_base.html">11261</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/ContextualMenu-mount.html">11323</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DefaultButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/DefaultButton-mount_base.html">750</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/DefaultButton-mount.html">730</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRow</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/DetailsRow-mount_base.html">2147</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/DetailsRow-mount.html">2174</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowFast</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/DetailsRowFast-mount_base.html">2208</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/DetailsRowFast-mount.html">2172</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>DetailsRowNoStyles</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/DetailsRowNoStyles-mount_base.html">1984</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/DetailsRowNoStyles-mount.html">1978</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Dialog</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Dialog-mount_base.html">2736</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Dialog-mount.html">2749</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>DocumentCardTitle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/DocumentCardTitle-mount_base.html">229</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/DocumentCardTitle-mount.html">225</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Dropdown</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Dropdown-mount_base.html">1949</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Dropdown-mount.html">1968</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusTrapZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/FocusTrapZone-mount_base.html">1125</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/FocusTrapZone-mount.html">1109</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>FocusZone</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/FocusZone-mount_base.html">1061</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/FocusZone-mount.html">1040</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/GroupedList-mount_base.html">40708</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/GroupedList-mount.html">41011</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/GroupedList-virtual-rerender_base.html">19710</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/GroupedList-virtual-rerender.html">19383</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedList</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/GroupedList-virtual-rerender-with-unmount_base.html">50162</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/GroupedList-virtual-rerender-with-unmount.html">49808</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/GroupedListV2-mount_base.html">223</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/GroupedListV2-mount.html">237</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/GroupedListV2-virtual-rerender_base.html">206</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/GroupedListV2-virtual-rerender.html">209</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>GroupedListV2</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/GroupedListV2-virtual-rerender-with-unmount_base.html">231</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/GroupedListV2-virtual-rerender-with-unmount.html">229</a></td>
            <td>2</td>
            <td></td>
           </tr>
<tr>
            <td>IconButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/IconButton-mount_base.html">1054</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/IconButton-mount.html">1057</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Label</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Label-mount_base.html">333</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Label-mount.html">336</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Layer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Layer-mount_base.html">2672</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Layer-mount.html">2728</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Link</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Link-mount_base.html">386</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Link-mount.html">393</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MenuButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/MenuButton-mount_base.html">957</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/MenuButton-mount.html">934</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>MessageBar</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/MessageBar-mount_base.html">21508</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/MessageBar-mount.html">21489</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Nav</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Nav-mount_base.html">1908</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Nav-mount.html">1926</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>OverflowSet</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/OverflowSet-mount_base.html">768</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/OverflowSet-mount.html">759</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Panel</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Panel-mount_base.html">1769</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Panel-mount.html">1775</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Persona</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Persona-mount_base.html">745</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Persona-mount.html">764</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>Pivot</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Pivot-mount_base.html">851</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Pivot-mount.html">832</a></td>
            <td>1000</td>
            <td></td>
           </tr>
<tr>
            <td>PrimaryButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/PrimaryButton-mount_base.html">828</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/PrimaryButton-mount.html">836</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Rating</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Rating-mount_base.html">4593</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Rating-mount.html">4629</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SearchBox</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/SearchBox-mount_base.html">896</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/SearchBox-mount.html">873</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Shimmer</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Shimmer-mount_base.html">1847</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Shimmer-mount.html">1837</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Slider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Slider-mount_base.html">1318</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Slider-mount.html">1317</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SpinButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/SpinButton-mount_base.html">2873</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/SpinButton-mount.html">2810</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Spinner</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Spinner-mount_base.html">375</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Spinner-mount.html">388</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SplitButton</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/SplitButton-mount_base.html">1822</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/SplitButton-mount.html">1822</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Stack</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Stack-mount_base.html">398</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Stack-mount.html">404</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithIntrinsicChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/StackWithIntrinsicChildren-mount_base.html">836</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/StackWithIntrinsicChildren-mount.html">858</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>StackWithTextChildren</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/StackWithTextChildren-mount_base.html">2607</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/StackWithTextChildren-mount.html">2607</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>SwatchColorPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/SwatchColorPicker-mount_base.html">6001</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/SwatchColorPicker-mount.html">6038</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TagPicker</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/TagPicker-mount_base.html">1419</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/TagPicker-mount.html">1422</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Text</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Text-mount_base.html">369</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Text-mount.html">369</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>TextField</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/TextField-mount_base.html">939</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/TextField-mount.html">924</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/ThemeProvider-mount_base.html">820</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/ThemeProvider-mount.html">830</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/ThemeProvider-virtual-rerender_base.html">585</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/ThemeProvider-virtual-rerender.html">586</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>ThemeProvider</td>
            <td>virtual-rerender-with-unmount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/ThemeProvider-virtual-rerender-with-unmount_base.html">1268</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/ThemeProvider-virtual-rerender-with-unmount.html">1241</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>Toggle</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Toggle-mount_base.html">634</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/Toggle-mount.html">609</a></td>
            <td>5000</td>
            <td></td>
           </tr>
<tr>
            <td>buttonNative</td>
            <td>mount</td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/buttonNative-mount_base.html">200</a></td>
            <td><a href="https://fluentuipr.z22.web.core.windows.net/pull/28157/perf-test/buttonNative-mount.html">190</a></td>
            <td>5000</td>
            <td></td>
           </tr></table></p></details>

<div id="perfComment9423"/>

**Review by khmakoto — APPROVED** [review]

**Pull request merged**